Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/PromptCodeReceiver.cs

```diff
@@ -41,7 +40,13 @@ public string RedirectUri
             var authorizationUrl = url.Build().ToString();
 
             Logger.Debug("Open a browser with \"{0}\" URL", authorizationUrl);
-            Process.Start(authorizationUrl);
+#if NETSTANDARD
+            Console.WriteLine("Please visit the following URL in a web browser, then enter the code:");
```

Comment:
How about

> Please visit the following URL in a web browser, then enter the code shown after authorization.

? (Do go ahead and merge before we get to the final wording, of course.)